src/list_of_files.cmake

```diff
@@ -0,0 +1,116 @@
+  ${CMAKE_CURRENT_SOURCE_DIR}/iw3pds.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/iw3unp29.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/ixgb.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/lengds.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/makwmo.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/mkfldsep.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/pdsens.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/pdseup.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/putgbe.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/putgben.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/putgbex.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/putgb.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/putgbn.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/q9ie32.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/r63w72.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/sbytec.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/sbyte.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/sbytesc.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/sbytes.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/skgb.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai00.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai01.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai08.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai15.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai18.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai19.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai24.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai38.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ai39.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3aq15.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3as00.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3difdat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3doxdat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi01.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi04.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi58.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi59.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi62.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi63.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi64.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi68.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi69.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi71.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi72.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi73.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi74.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi75.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi76.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi82.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi83.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi88.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fi92.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fp11.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fp12.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fp13.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fs13.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fs15.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fs21.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3fs26.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ft32.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3kind.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3locdat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3movdat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3nogds.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3pradat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3reddat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3tagb.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3trnarg.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3unpk77.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3utcdat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3valdat.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/w3ymdh4.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/xmovex.f
+  ${CMAKE_CURRENT_SOURCE_DIR}/xstore.f
```

Comment:
`list_of_files.cmake` is in the same directory as `src`. `CMAKE_CURRENT_SOURCE_DIR` is not necessary.
Please remove.